cartoframes/viz/map.py

```diff
@@ -274,23 +274,23 @@ def get_content(self):
             '_airship_path': self._airship_path
         }
 
-    def publish(self, name, table_name=None, credentials=None, password=None):
+    def publish(self, name, password, table_name=None, credentials=None):
         """Publish the map visualization as a CARTO custom visualization (aka Kuviz).
 
         Args:
-            name (str): The Kuviz name on CARTO
+            name (str): The Kuviz name on CARTO.
+            password (str): By setting it, your Kuviz will be protected by
+                password. When someone tries to show the Kuviz, the password
+                will be requested. To disable password you must set it to None.
```

Comment:
I haven't reviewed this before, but I'm unsure if referring to the term `Kuviz` makes sense. To me it looks like an internal term. We are not adding any reference to know more about what a Kuviz is neither, so maybe we could just use `visualization` or `map`. What do you think?

Reply:
Yes, visualization  :+1: